packages/codemods/rollup.config.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import commonConfig from '../../utils/build/rollup.config.js';
+
+export default {
+  ...commonConfig[0],
+  output: [
+    commonConfig[0].output[0],
+    {
+      ...commonConfig[0].output[1],
+      entryFileNames: '[name].mjs'
```

Comment:
It's 2024 and the CommonJS / ESM struggles are still real! 😡 


Currently, jscodeshift [doesn't support ESM](https://github.com/facebook/jscodeshift/issues/500#:~:text=This%20has%20made%20the%20jscodeshift%20project%20far%20more%20difficult%20to%20regularly%20maintain%20for%20everyone.%20There%20is%20also%20the%20matter%20that%20jscodeshift%20in%20its%20current%20state%20is%20(as%20far%20as%20I%27m%20aware)%20permanently%20incompatible%20with%20ESM.), and the latest version of libraries like [Chalk](https://github.com/chalk/chalk#:~:text=Chalk%205%20is%20ESM.%20If%20you%20want%20to%20use%20Chalk%20with%20TypeScript%20or%20a%20build%20tool%2C%20you%20will%20probably%20want%20to%20use%20Chalk%204%20for%20now.) and [@inquire/prompts](https://github.com/SBoudrias/Inquirer.js/issues/999) dropped CommonJS support. 

After trying several approaches, I’ve found a workable solution:
- Setting "type": "module" did not resolve the issues with jscodeshift.
- Directing the "module" to the CommonJS build failed with Chalk and other ESM-only libraries.

Since this package requires Node.js version 18 or higher, we can use the .mjs extension to maintain compatibility and avoid encountering ERR_REQUIRE_ESM errors when executing the script.

I'm open to suggestions for reusing the common Rollup config instead.